gorums.proto

```diff
@@ -14,6 +15,17 @@ extend google.protobuf.MethodOptions {
   bool qc_future = 50004;
   bool qf_with_req = 50005;
   bool per_node_arg = 50006;
+  bool strict_ordering = 50007;
 
   string custom_return_type = 51000;
 }
+
+service Gorums {
+  rpc StrictOrdering(stream Message) returns (stream Message);
+}
+
+message Message {
```

Comment:
So, since this is probably moved into `dev` (i.e. made part of the Gorums generated code), maybe it would be good to name this something else too, e.g. `Message` --> `GorumsMessage` or `InternalMessage` or something better...